### PR TITLE
Static Page

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,3 +14,14 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+//= require jquery
```
Comment (yongsokheng) [inline app/assets/javascripts/application.js]: duplicate code

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -13,3 +13,21 @@
  *= require_tree .
  *= require_self
  */
+
```
Comment (yongsokheng) [inline app/assets/stylesheets/application.css.scss]: i think should create custom.scss, and remove these codes to this file.

File: app/views/shared/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div class="footer">
+  <p class="text-center">
+    SKCK_SYSTEM
```
Comment (yongsokheng) [inline app/views/shared/_footer.html.erb]: i18n

File: app/views/shared/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <span class="navbar-brand"><%= link_to "SKCK SYSTEM", root_path %></span>
```
Comment (yongsokheng) [inline app/views/shared/_header.html.erb]: i18n

File: app/views/shared/_footer.html.erb (hunk 3 above)
Comment (NgengChhengkim) [inline app/views/shared/_footer.html.erb]: chua viet trong i18n vi chua tim dc tem hop ly. do phai su lai nhieu lan

**Pull request merged**